cabal-install/Distribution/Client/Init.hs

```diff
@@ -91,19 +91,28 @@ import Distribution.Simple.PackageIndex
 import Distribution.Text
   ( display, Text(..) )
 
+import Distribution.Client.PackageIndex
+  ( elemByPackageName )
+import Distribution.Client.IndexUtils
+  ( getSourcePackages )
+import Distribution.Client.Types
+  ( SourcePackageDb(..), Repo )
+
 initCabal :: Verbosity
```

Comment:
might be good to add a Haddock comment here to spell out what the api does! (it didnt have that before i guess, so while you're changing it is probably a good time to add some docs!)